kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerServerException.java

```diff
@@ -35,23 +35,37 @@ public class SpinnakerServerException extends SpinnakerException {
    */
   private final String rawMessage;
 
+  /* */
+  /* retrofit's response body derived as map or null if response not provided or custom message has been provided */
+  /*
+    private final Map<String, Object> responseBody;
+  */
   /**
-   * Parses the message from the {@link RetrofitErrorResponseBody} of a {@link RetrofitError}.
+   * Parse the response body of {@link RetrofitError} as json into responseBody and extract the
```

Comment:
With the code the way it is, mentioning RetrofitErrorResponseBody and RetrofitError seems more correct.  
```suggestion
   * Parse the message from the {@link RetrofitErrorResponseBody} of a {@link RetrofitError}.
```